examples/full-elastic-stack/k8s/ValidatingWebhookConfiguration.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration # Can also be a MutatingWebhookConfiguration if required
+metadata:
+  name: kube-audit-rest
+  labels:
+    app: kube-audit-rest
+webhooks:
+  - name: kube-audit-rest.example-kube-audit-rest.svc.cluster.local
+    failurePolicy: Ignore 
+    timeoutSeconds: 1 
+    sideEffects: None
+    clientConfig:
+      service:
+        name: kube-audit-rest
+        namespace: example-kube-audit-rest
+        path: "/log-request"
+      caBundle: "$CABUNDLEB64" # To be replaced
+    rules: # To be reduced as needed
```

Comment:
I would add that this should be reduced to at least not capture k8s secrets and that any contents inside configmas will be captured too

Reply:
Is `It will record all create/mutation/deletion API calls, which can leak service account tokens via secrets. This is to show to maximum capabilities.` in the readme not enough? Want to give folks the option while experimenting to see everything this can do